build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -1107,8 +1125,12 @@ CONFIG_DEBUG_VERBOSE=y
 #
 # Subsystem Debug Options
 #
-# CONFIG_DEBUG_BINFMT is not set
-# CONFIG_DEBUG_BINMGR is not set
+CONFIG_DEBUG_BINFMT=y
```

Comment:
Why did you enable these?

Reply:
I think we can enable when debugging, if needed. It affects the time, like loading time, we'd better disable.

Reply:
Okay, I disabled.